src/app/components/admin/page-admin-articles/page-admin-article-edit/page-admin-article-edit.component.html

```diff
@@ -0,0 +1,13 @@
+<ksi-admin-sidebar></ksi-admin-sidebar>
+
+<div class="content-container">
+    <div class="content">
+        <h2>Work in progress...</h2>
+        <a [routerLink]="['/', routes.routes.admin._, routes.routes.admin.waves._]"> Zpět na přehled</a>
+
+        <div class="">
+            <img src="https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExaWxremdsOW1mcmJtY3hxZ3BieTczZWN3ODB5MW8zY3BkcDVzeHl4ZyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/VHOF8pfPZOt9p018zw/giphy.gif"
```

Comment:
je mi to líto, je to krásný gif, ale poprosím tě, abys nepoužíval externí obrázky. Externí obrázky zpomalují stránku a prozrazují tvoji IP adresu a fakt, že jsi na KSI webu ostatním službám.